src/Termonad/Types.hs

```diff
@@ -408,6 +408,11 @@ data ConfigOptions = ConfigOptions
     -- If 'False', then bold can be applied separately to colors from both the
     -- 'Termonad.Config.Colour.BasicPalatte' and
     -- 'Termonad.Config.Colour.ExtendedPalatte'.
+  , enableSixel :: !Bool
+    -- ^ Enable SIXEL to draw graphics in a terminal.
+    --
+    -- Note that SIXEL may not be fully supported in VTE. Follow
+    -- <https://gitlab.gnome.org/GNOME/vte/-/issues/253> for more information.
```

Comment:
@gelisam I added this message here that I think at least makes it somewhat clear to users that Termonad doesn't guarantee how well SIXEL will work, even if you enable it.

Reply:
Thanks!